src/components/filter/Filter.tsx

```diff
@@ -121,7 +122,13 @@ export const Filter = ({ filterTypes, initialFilters, setQueryVariables }: Props
           <Button
             disabled={isNoFilterSet}
             title={texts.filter.resetFilter}
-            onPress={() => setFilters(initialFilters)}
+            onPress={() => {
+              setIsCollapsed(true);
+
+              setTimeout(() => {
+                setFilters(initialFilters);
+              }, 500);
+            }}
```

Comment:
how does this apply to sorting?

Reply:
This has nothing to do with the sorting. But I didn't want to create a new PR for it. It just makes the filter section close automatically if you press the reset filter button.